### ISSUE TITLE
No authentication documentation

### ISSUE THREAD

**Issue by SebastianKG** [JetBrains/kotless#53]
I am using the Ktor interface for Kotless; I was wondering about how to handle authentication; say through JWT tokens, or even connecting up with Auth0. I don't see any documentation about this--I assume Kotless will not inspect Ktor's authentication middleware, but I'm not sure either way. There isn't any documentation regarding adding on API gateway lambda authorizers, either, though I suppose that could be done in the Terraform extension points & by defining the authorizer with the rest of the API.

Any pointers about this would be great; I think this is probably a common concern so it may fit well in the README/Wiki as well. Thanks; this is an amazing toolkit and has worked flawlessly thus far.

**Comment by TanVD** [comment]
Yep, current situation is that authorizers are not supported. But we are willing to support it as soon as possible.
Right now you can either implement on your own, via integration with DynamoDB (we use such approach in a few internal JetBrains applications). Other approach is to implement it via Terraform extensions.
